lib/rack/cache/version.rb

```diff
@@ -1,5 +1,5 @@
 module Rack
   module Cache
-    VERSION = '1.15.0'
+    VERSION = '1.15.1'
```

Comment:
Avoid release management changes in other PRs. Tools will do this for us, later.

```suggestion
    VERSION = '1.15.0'
```

Reply:
https://github.com/rack/rack-cache/pull/22/commits/70282c9eedd182a74df78ea198b24ffe88ab9b4f